src/main.cpp

```diff
@@ -342,6 +353,18 @@ int main(int argc, char *argv[])
     if (ksrc != "")
       extra_flags = get_kernel_cflags(utsname.machine, ksrc, kobj);
   }
+
+  for (auto dir : include_dirs)
+  {
+    extra_flags.push_back("-isystem");
```

Comment:
Shouldn't this just be `-I`?

Reply:
Keeping consistent with other includes we add via flags (as per https://github.com/iovisor/bpftrace/pull/657), although it might make sense to use -I instead here, to keep behavior consistent with clang. Do we want to have `-I` and `-isystem` in bpftrace?

Reply:
Those ones are a special case because they are system headers. Regular ones that people specify with `-I` in bpftrace can just be passed through as `-I` unless that's breaking things here too?